docker/next-app/Dockerfile.next-app

```diff
@@ -19,4 +19,10 @@ COPY --from=builder /app/package.json /app/
 
 EXPOSE 3000
 
+# container would not receive SIGTERM from docker when shutting down so Docker would force kill
+# the container after 10s.  This will make shutdown faster because the SIGTERM will be handled appropriately.
+# https://maximorlov.com/process-signals-inside-docker-containers/
+RUN apk add --no-cache tini
+ENTRYPOINT ["/sbin/tini", "--"]
```

Comment:
thank you!  The next-app container was always the last to finish in `docker compose down`